doc/classes/AcceptDialog.xml

```diff
@@ -76,6 +76,7 @@
 	<signals>
 		<signal name="cancelled">
 			<description>
+			Emitted when the dialog is closed or the button created with [method add_cancel] is pressed.
```

Comment:
The indentation is incorrect here, it should be intended one tab further. Might be worth melding this commit into the next one too as the second commit fixes it up and includes most of the content from this PR.

Reply:
done.